scripts/release-tools/clusters.sh

```diff
@@ -8,7 +8,8 @@ ARTIFACTS_DIR="artifacts"
 main() {
   local action="${1}"
   local cluster_prefix="${2:-os4-9-demo}"
```

Comment:
The `cluster_prefix` will be problematic here, as it supports only OpenShift and we now have both OS and GKE.

For GKE, you actually hardcoded prefix as `qa-demo`. For OS, we have `os4-9-demo`. I think that hardcoding is fine when we create clusters in `create_rc_openshift_cluster` and `create_qa_gke_cluster`. 

However, we also use it for `generate_slack_message` and `merge_kubeconfigs` and there, we would need to differentiate there which cluster we mean: OS or GKE.

I would propose the following (as probably the simplest approach):
- Remove `cluster_prefix` from `exec_option` and `main`
- Hardcode `cluster_prefix` in  `create_rc_openshift_cluster` and `create_qa_gke_cluster`. 
- Enhance `merge_kubeconfigs` option, so that we cover OS and GKE clusters (we can simply call this function twice in `exec_option`
- Duplicate `generate_slack_message`, so that we have 2 options in the menu: one for OS, one for GKE.


What do you think?

Reply:
Sounds good. Done